CHANGELOG.md

```diff
@@ -11,8 +11,11 @@ All notable changes to this project will be documented in this file.
 N/A
 
 ### Enhancements
-N/A
-
+- New **NSAttributedString** extensions
```

Comment:
btw, I think we can just use 1 hyperlink to the PR per section. For example:
If you add 3 array extensions and 2 string extensions, just reference the PR once under Array and once under String. I think it will be cleaner and help keep CHANGELOG entries on one line for readability.